pkg/reconciler/kubernetes/tektoninstallerset/tektoninstallerset.go

```diff
@@ -166,6 +166,15 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, installerSet *v1alpha1.T
 	// Update Ready status of Controller
 	installerSet.Status.MarkControllerReady()
 
+	// job
+	err = installer.IsJobCompleted()
+	if err != nil {
+		job := installer.Manifest.Filter(mf.ByKind("Job")).Resources()[0]
+		labels := installerSet.GetLabels()
+		logger.Info("job not ready in installerset, name: %s, created-by: %s, in namespace: %s", installerSet.GetName(), labels[v1alpha1.CreatedByKey], job.GetNamespace())
+		return err
```

Comment:
this will work but let us move this to https://github.com/tektoncd/operator/blob/c6c96e1503dbc31f52a64e14e0f285af7624f476/pkg/reconciler/kubernetes/tektoninstallerset/install.go#L368

Doing so will make it easy to report on each job in the installerset instead of focusing on Job[0].

Reply:
on this line (368) we still need to return the error, but add the log right before it.

Reply:
I have updated the patch could you please take a look at it once 🙂